pages/index.tsx

```diff
@@ -216,3 +222,13 @@ export default function Home() {
     </main>
   )
 }
+
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+
+export async function getStaticProps({ locale }) {
+  return {
+    props: {
+      ...(await serverSideTranslations(locale)),
```

Comment:
Just a reminder that we should double-check that this isn't using too much Vercel budget, or has some reasonable caching enabled to avoid that. I'm not opposed to it though

Reply:
Yeah good call, I wanted to first get things running and then ideally make them only run on the client.